toolkit/docs/building/building.md

```diff
@@ -132,6 +134,7 @@ A set of bootstrapped toolchain packages (gcc etc.) are used to build CBL-Marine
 ```bash
 # Populate Toolchain from pre-existing binaries
 sudo make toolchain REBUILD_TOOLS=y
+sudo make copy-toolchain-rpms
```

Comment:
Btw. @dmcilvaney is currently working on toolkit updates, which might make this step unnecessary and simplify the build process. No need to change anything in this PR, though, I mostly wanted to let Daniel know, that the docs might require some update after his improvements are done.

Reply:
acknowledged